model/TransferResponseV2.ts

```diff
@@ -0,0 +1,43 @@
+import { Currency, Date } from "isoly"
+import { BookingInfoResponse } from "./BookingInfoResponse"
+import { FundingAccountResponseV2 } from "./FundingAccountResponseV2"
+import { ProviderCode } from "./ProviderCode"
+import { TransferDestinationResponse } from "./TransferDestinationResponse"
+import { TransferStatus } from "./TransferStatus"
+
+export interface TransferResponseV2 {
+	providerCode: ProviderCode
+	providerTransferId: string
+	amount: number
+	currency: Currency
+	status: TransferStatus
+	errorMessage?: string
+	createdDate: Date
+	paymentDate?: Date
+	reference: string
+	source: FundingAccountResponseV2
+	destination: TransferDestinationResponse
+	bookingInfo?: BookingInfoResponse
+	createdBy: string
+}
+
+export namespace TransferResponseV2 {
+	export function is(value: TransferResponseV2 | any): value is TransferResponseV2 {
+		return (
+			typeof value == "object" &&
+			ProviderCode.is(value.providerCode) &&
+			typeof value.providerTransferId == "string" &&
+			typeof value.amount == "number" &&
+			Currency.is(value.currency) &&
+			TransferStatus.is(value.status) &&
+			(value.errorMessage == undefined || typeof value.errorMessage == "string") &&
+			Date.is(value.createdDate) &&
+			(value.paymentDate == undefined || Date.is(value.paymentDate)) &&
+			typeof value.reference == "string" &&
+			FundingAccountResponseV2.is(value.source) &&
+			TransferDestinationResponse.is(value.destination) &&
+			(value.bookingInfo == undefined || BookingInfoResponse.is(value.bookingInfo)) &&
+			typeof value.createdBy == "string"
+		)
+	}
```

Comment:
this is wrong for the non-paxpay transfers

Reply:
made a summary type